libs/service/service.go

```diff
@@ -30,9 +30,6 @@ type Service interface {
 	// Return true if the service is running
 	IsRunning() bool
 
-	// Quit returns a channel, which is closed once service is stopped.
-	Quit() <-chan struct{}
```

Comment:
If I'm understanding correctly, this was just removed as a possible method that a service could override, and instead is just a signal provided to a service by baseapp. This seems reasonable since I can't think of cases where you would need to override `Quit` and it seems like we never did.

Reply:
Quit, was just the "channel form of Wait()" and we weren't implementing it anywhere and also not really using it anywhere (almost). 

Like most of the service method, the `BaseService` was the only ever implementation most of the methods, and I think most services *shouldn't* have to implement their own lifecycle methods.

Reply:
Reading your response [here](https://github.com/tendermint/tendermint/pull/7292#issuecomment-974060120), it looks like our north star is to have some external orchestrator for `Service` lifecycle which I'm excited for.